src/backend/gpopt/translate/CTranslatorDXLToPlStmt.cpp

```diff
@@ -602,6 +602,10 @@ CTranslatorDXLToPlStmt::TranslateDXLTblScan(
 		plan->qual = qual;
 	}
 
+	if (md_rel->IsAORowOrColTable())
```

Comment:
do we have checked the `ao_column`?

Reply:
ao_column checked in IsAORowOrColTable()